ci-operator/step-registry/agent/e2e/generic/agent-e2e-generic-workflow.yaml

```diff
@@ -5,10 +5,12 @@ workflow:
     pre:
       - chain: agent-pre
     test:
-      - chain: agent-test
+      - chain: agent-test-conformance
```

Comment:
I think that the generic workflow it's used in some other presubmit jobs, so either:
1) Define a new generic-conformance workflow
2) Keep the generic workflow but override the test chain directly in the release jobs